scrapi/base/__init__.py

```diff
@@ -211,12 +211,9 @@ def get_properties(self, result, property_list):
             prop.extend(result.xpath(
                 '//ns0:{}/node()'.format(item),
                 namespaces=self.NAMESPACES
-            ) or [''])
+            ))
 
-            if len(prop) > 1:
-                properties[item] = [self.copy_to_unicode(item) for item in prop]
-            else:
-                properties[item] = self.copy_to_unicode(prop[0])
+            properties[item] = [self.copy_to_unicode(element) for element in prop]
```

Comment:
So `properties[item]` will always be a list?
Will this break the current elasticsearch mappings?